libs/components/src/modal/modal.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, Input } from "@angular/core";
+
+@Component({
+  selector: "bit-modal",
+  templateUrl: "./modal.component.html",
+})
+export class ModalComponent {
+  // Controls the size of the bit modal displayed. Needs to be implemented.
+  @Input()
+  modalSize: "small" | "default" | "large";
```

Comment:
Lets keep these on the same line.

```suggestion
  @Input() modalSize: "small" | "default" | "large";
```